### PR TITLE
chore/switch to sapphire

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

File: src/events/onceReady.js [hunk 1]
```diff
@@ -3,42 +3,39 @@ const { Team } = require('discord.js');
...
 
 	async run() {
+		const { client } = this.context;
 		try {
-			await this.client.fetchApplication();
+			await this.context.client.fetchApplication();
 		} catch (err) {
 			if (++retries === 3) return process.exit();
-			this.client.emit('warning', `Unable to fetchApplication at this time, waiting 5 seconds and retrying. Retries left: ${retries - 3}`);
+			client.emit('warning', `Unable to fetchApplication at this time, waiting 5 seconds and retrying. Retries left: ${retries - 3}`);
```
Comment (vladfrangu) [inline src/events/onceReady.js]: isn't that subtraction reversed?

File: src/events/verbose.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ const { Event } = require('klasa');
...
 module.exports = class extends Event {
 	run(log) {
-		this.client.console.verbose(log);
+		this.context.client.console.verbose(log);
 	}
 
-	init() {
-		if (!this.client.options.consoleEvents.verbose) this.disable();
+	onLoad() {
+		this.enabled = this.context.client.options.consoleEvents.verbose;
```
Comment (vladfrangu) [inline src/events/verbose.js]: Why are some onLoad and some in the constructor

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: src/events/onceReady.js (hunk 1 above)
Comment (kyranet) [reply src/events/onceReady.js]: wdym?

Comment (vladfrangu) [reply src/events/onceReady.js]: retries - 3 would yield -2, -1, then 0

**Review by vladfrangu — APPROVED** [review]

File: src/events/onceReady.js (hunk 1 above)
Comment (kyranet) [reply src/events/onceReady.js]: That's Klasa for you, I didn't write that LOL

**Review by favna — APPROVED** [review]

**Pull request merged**